### PR TITLE
Update dbus to 0.6 and bump version

### PR DESCRIPTION
**Pull request by Eijebong**
cc @nox

### CODE REVIEW

File: src/bluetooth_gatt_characteristic.rs [hunk 1]
```diff
@@ -91,12 +90,13 @@ impl BluetoothGATTCharacteristic {
...
-                                    Box::new(MessageItem::Variant(Box::new(o.into()))))],
-                    None => vec![],
-                },
-                Cow::Borrowed("{sv}")
+                MessageItemArray::new(
+                    match offset {
+                        Some(o) => vec![MessageItem::DictEntry(Box::new("offset".into()),
+                                         Box::new(MessageItem::Variant(Box::new(o.into()))))],
+                        None => vec![]
+                    }, Signature::from("{sv}")
```
Comment (dati91) [inline src/bluetooth_gatt_characteristic.rs]: `{sv}` -> `a{sv}`

File: src/bluetooth_gatt_characteristic.rs [hunk 2]
```diff
@@ -121,12 +121,13 @@ impl BluetoothGATTCharacteristic {
...
```
Comment (dati91) [inline src/bluetooth_gatt_characteristic.rs]: `{sv}` -> `a{sv}`

File: src/bluetooth_gatt_descriptor.rs [hunk 3]
```diff
@@ -80,12 +79,13 @@ impl BluetoothGATTDescriptor {
...
-                                    Box::new(MessageItem::Variant(Box::new(o.into()))))],
-                    None => vec![],
-                },
-                Cow::Borrowed("{sv}")
+                MessageItemArray::new(
+                    match offset {
+                        Some(o) => vec![MessageItem::DictEntry(Box::new("offset".into()),
+                                         Box::new(MessageItem::Variant(Box::new(o.into()))))],
+                        None => vec![]
+                    }, Signature::from("{sv}")
```
Comment (dati91) [inline src/bluetooth_gatt_descriptor.rs]: `{sv}` -> `a{sv}`

File: src/bluetooth_gatt_descriptor.rs [hunk 4]
```diff
@@ -110,12 +110,13 @@ impl BluetoothGATTDescriptor {
...
```
Comment (dati91) [inline src/bluetooth_gatt_descriptor.rs]: `{sv}` -> `a{sv}`

**Review by dati91 — CHANGES_REQUESTED** [review]
Hi, and thanks for the PR.

With the current code it fails to read or write values, because of the wrong signature.

File: src/bluetooth_gatt_descriptor.rs [hunk 5]
```diff
@@ -110,12 +110,13 @@ impl BluetoothGATTDescriptor {
...
-                match offset {
-                    Some(o) => vec![MessageItem::DictEntry(Box::new("offset".into()),
-                                    Box::new(MessageItem::Variant(Box::new(o.into()))))],
-                    None => vec![],
-                },
-                Cow::Borrowed("{sv}")
+                MessageItemArray::new(
+                    match offset {
+                        Some(o) => vec![MessageItem::DictEntry(Box::new("offset".into()),
+                                         Box::new(MessageItem::Variant(Box::new(o.into()))))],
```
Comment (dati91) [inline src/bluetooth_gatt_descriptor.rs]: indentation

File: src/bluetooth_gatt_descriptor.rs [hunk 6]
```diff
@@ -80,12 +79,13 @@ impl BluetoothGATTDescriptor {
...
```
Comment (dati91) [inline src/bluetooth_gatt_descriptor.rs]: indentation

File: src/bluetooth_gatt_characteristic.rs [hunk 7]
```diff
@@ -121,12 +121,13 @@ impl BluetoothGATTCharacteristic {
...
-                match offset {
-                    Some(o) => vec![MessageItem::DictEntry(Box::new("offset".into()),
-                                    Box::new(MessageItem::Variant(Box::new(o.into()))))],
-                    None => vec![],
-                },
-                Cow::Borrowed("{sv}")
+                MessageItemArray::new(
+                    match offset {
+                        Some(o) => vec![MessageItem::DictEntry(Box::new("offset".into()),
+                                         Box::new(MessageItem::Variant(Box::new(o.into()))))],
```
Comment (dati91) [inline src/bluetooth_gatt_characteristic.rs]: indentation

File: src/bluetooth_gatt_characteristic.rs [hunk 8]
```diff
@@ -91,12 +90,13 @@ impl BluetoothGATTCharacteristic {
...
```
Comment (dati91) [inline src/bluetooth_gatt_characteristic.rs]: indentation

File: src/bluetooth_gatt_characteristic.rs (hunk 1 above)
Comment (Eijebong) [reply src/bluetooth_gatt_characteristic.rs]: Woups

**Comment by Eijebong** [comment]
There, fixed :)

**Comment by dati91** [comment]
Great, thanks!

**Pull request merged**